.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
Always better to specify exact versions, to avoid surprises later. I use `ubuntu-18.04` in all other files.

Reply:
good idea